cmd/influxd/run/server_test.go

```diff
@@ -1010,6 +1010,9 @@ func NewConfig() *run.Config {
 	c.HTTPD.Enabled = true
 	c.HTTPD.BindAddress = "127.0.0.1:0"
 	c.HTTPD.LogEnabled = testing.Verbose()
+
+	c.UDP.BindAddress = "127.0.0.1:0"
+	c.UDP.Database = "db0"
```

Comment:
I think the real problem here is that the server isn't observing if UDP is enabled, and actually adding it to the services.  I'm guessing a couple of our endpoints have that problem.  I'll merge this, and then do another PR that fixes services starting that aren't enabled.